### ISSUE TITLE
Animated stickers?

### ISSUE THREAD

**Issue by fabiosoft** [zhukov/webogram#1874]

**Comment by SigHunter** [comment]
desperately need!

it sucks that telegram webclient is so dead since 2 years :(

**Comment by azzimuth** [comment]
Guys, this project is dead...

**Comment by abitrolly** [comment]
So what is needed to get rid of this message?

> This message is currently not supported on Telegram Web.

**Comment by azzimuth** [comment]
> So what is needed to get rid of this message?
> 
> > This message is currently not supported on Telegram Web.

Implementation of animated stickers. But I'd rather start using Telegram Desktop. The new features appear much faster there.

**Comment by fabiosoft** [comment]
If you need a web based client try: https://evgeny-nadymov.github.io/telegram-react/
Frequently updated and supports almost all mobile client features (stickers included ;) )

**Comment by abitrolly** [comment]
I thought Telegram Desktop is just an Electron wrapper over this web app. Or is it a native application?

**Comment by azzimuth** [comment]
> I thought Telegram Desktop is just an Electron wrapper over this web app. Or is it a native application?

No, it's a separate application, developed in C++. See here https://github.com/telegramdesktop/tdesktop

**Comment by abitrolly** [comment]
@fabiosoft I am a bit of paranoid about using non-official clients. :D

**Comment by Aokromes** [comment]
> @fabiosoft I am a bit of paranoid about using non-official clients. :D

that's official too.

**Comment by abitrolly** [comment]
@Aokromes it is possible to move it to official domain then? Like https://react.telegram.org or https://web2.telegram.org

**Comment by Aokromes** [comment]
> @Aokromes it is possible to move it to official domain then? Like [react.telegram.org](https://react.telegram.org) or [web2.telegram.org](https://web2.telegram.org)

ask on twiteer :)

**Comment by abitrolly** [comment]
Is Twitter an official support channel? :D

**Comment by Aokromes** [comment]
it's official way to cry for features xd

**Comment by abitrolly** [comment]
@Aokromes sounds very encouraging. :D

**Comment by abitrolly** [comment]
![image](https://user-images.githubusercontent.com/8781107/116215868-bb81cb80-a750-11eb-9b5b-859f2fa77aac.png)

Looks like this version is officially dead. :D

**Comment by fabiosoft** [comment]
What a pity... i use a public/shared PC where i cannot install new software, so web version is very useful

**Comment by abitrolly** [comment]
@fabiosoft https://evgeny-nadymov.github.io/telegram-react/ might still work better.

**Comment by fabiosoft** [comment]
> @fabiosoft https://evgeny-nadymov.github.io/telegram-react/ might still work better.

i know, thanks :)

**Comment by azzimuth** [comment]
@fabiosoft, Telegram announced [two new Web versions](https://telegram.org/blog/payments-2-0-scheduled-voice-chats#new-web-versions).

**Comment by lutzky** [comment]
It appears that a new web version is now marked as official. https://telegram.org/apps now links to https://webk.telegram.org/. It seems very similar to the evgeny-nadymov version (albeit not completely identical - for one thing the hamburger menu on this version explicitly says "webk version" whereas the evgeny-nadymov version does not) - and indeed it supports animated stickers.

**Comment by abitrolly** [comment]
I am glad I am commented here. :D

If `webk` Evgeny's, thebw here https://webz.telegram.org/ comes from?
